### PR TITLE
Update docs

### PR DESCRIPTION
**Pull request by betarelease**
## Description

This PR fixes links to use path starting with docs root 
so that the links do not break

## PR Checklist

- [x] I've read the [contributing guidelines](https://github.com/pyrsia/.github/blob/main/contributing.md).
- [x] I've read ["What is a Good PR?"](https://github.com/pyrsia/pyrsia/blob/main/docs/community/get_involved/good_pr.md)
- [x] I've included a good title and brief description along with how to review them.
- [x] I've linked any associated an [issue](https://github.com/pyrsia/pyrsia/issues).

### CODE REVIEW

File: docs/developers/pyrsia-with-docker.md [hunk 1]
```diff
@@ -4,7 +4,7 @@ sidebar_position: 5
 
 # Pyrsia and Docker
 
-Once you have setup your [local environment](../get_involved/local_dev_setup.md) with Pyrsia you are now ready to run Pyrsia with Docker support.
+Once you have setup your [local environment](docs/community/get_involved/local_dev_setup.md) with Pyrsia you are now ready to run Pyrsia with Docker support.
```
Comment (erwin1) [inline docs/developers/pyrsia-with-docker.md:7]: shouldn't this start with `/` ?
```suggestion
Once you have setup your [local environment](/docs/community/get_involved/local_dev_setup.md) with Pyrsia you are now ready to run Pyrsia with Docker support.
```

Comment (betarelease) [reply docs/developers/pyrsia-with-docker.md:7]: It worked without the slash on my machine. But I think for consistency it is better with /

**Review by fishseabowl — APPROVED** [review]

**Review by dasmanas — APPROVED** [review]
lg

**Review by erwin1 — APPROVED** [review]

**Review by erwin1 — APPROVED** [review]

**Comment by erwin1** [comment]
@betarelease I merged the other 2 doc PRs and fixed conflicts in this one, but as it turned out, you had the same changes already in one of the other PRs.

**Review by fishseabowl — APPROVED** [review]

**Comment by betarelease** [comment]
Yeah it is ok to close this PR now. This was the first one to be merged actually. In any case I will close this.

**Pull request closed without merging**